lib/api.js

```diff
@@ -56,7 +64,7 @@ export function trackSave(id, promise) {
 export async function getType(id) {
     if (!id) return null
     await (tracking[id] || dummy)
-    return await get(id, true, false, 'server')
+    return await get(id, true, false, 'server-preferred')
```

Comment:
minor: can import common/offline-data-service/constants.json and use SERVER_PREFERRED as well